packages/legend-query/src/components/QueryBuilderResultPanel.tsx

```diff
@@ -212,33 +212,30 @@ export const QueryBuilderResultPanel = observer(
               </div>
             )}
             <button
-              className="panel__header__action"
+              className="panel__header__action query-builder__result__execute-btn"
               onClick={execute}
               tabIndex={-1}
               title="Execute"
             >
-              <PlayIcon />
-            </button>
-            <button
-              className="panel__header__action"
-              onClick={executePlan}
-              tabIndex={-1}
-              title="View Exection Plan"
-            >
-              <PaperScrollIcon />
+              <div className="query-builder__result__execute-btn__icon">
+                <PlayIcon />
+              </div>
+              <div className="query-builder__result__execute-btn__label">
+                Execute
```

Comment:
Can we inlcude video/screenshot of this update in the PR ?

Reply:
sure thing!